docs/dunst.1.pod

```diff
@@ -59,7 +59,7 @@ Display a notification on startup.
 
 =head1 CONFIGURATION
 
-A default configuration file is included (usually ##SYSCONFDIR##/dunst/dunstrc)
```

Comment:
Why do you replace this here with a variable and not let it be the build time constant?

Reply:
@pliski could you take a look at this comment?